code/modules/mob/living/silicon/mommi/mommi_modules.dm

```diff
@@ -96,3 +92,21 @@
 		"Gravekeeper" = "cogspider"
 		)
 	law_type = "Gravekeeper"
+
+/obj/item/weapon/robot_module/mommi/sammi
+	name = "stationary assembly mmi robot module"
+
+/obj/item/weapon/robot_module/mommi/sammi/New(var/mob/living/silicon/robot/R)
+	..()
+	modules = new/list()
```

Comment:
what

Reply:
I think it resets the module list while still inheriting New()